src/current/v23.2/molt-fetch.md

```diff
@@ -0,0 +1,413 @@
+---
+title: MOLT Fetch
+summary: Learn how to use the MOLT Fetch tool to move data from a source database to CockroachDB.
+toc: true
+docs_area: migrate
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+MOLT Fetch moves data from a source database into CockroachDB as part of a [database migration]({% link {{ page.version.version }}/migration-overview.md %}).
+
+MOLT Fetch can use `IMPORT INTO` or `COPY FROM` to move the source data to CockroachDB via cloud storage (Google Cloud Storage or Amazon S3), a local file server, or directly without an intermediate store. For details, see [Usage](#usage).
+
+## Supported databases
+
+The following databases are currently supported:
+
+- [PostgreSQL]({% link {{ page.version.version }}/migrate-from-postgres.md %})
+- [MySQL]({% link {{ page.version.version }}/migrate-from-mysql.md %})
+- CockroachDB
+
+## Installation
+
+To install MOLT Fetch, download the binary that matches your system. To download the latest binary:
+
+| Operating System |                                    AMD 64-bit                                   |                                    ARM 64-bit                                   |
+|------------------|---------------------------------------------------------------------------------|---------------------------------------------------------------------------------|
+| Windows          | [Download](https://molt.cockroachdb.com/molt/cli/molt-latest.windows-amd64.tgz) | [Download](https://molt.cockroachdb.com/molt/cli/molt-latest.windows-arm64.tgz) |
+| Linux            | [Download](https://molt.cockroachdb.com/molt/cli/molt-latest.linux-amd64.tgz)   | [Download](https://molt.cockroachdb.com/molt/cli/molt-latest.linux-arm64.tgz)   |
+| Mac              | [Download](https://molt.cockroachdb.com/molt/cli/molt-latest.darwin-amd64.tgz)  | [Download](https://molt.cockroachdb.com/molt/cli/molt-latest.darwin-arm64.tgz)  |
+
+For previous binaries, see the [MOLT version manifest](https://molt.cockroachdb.com/molt/cli/versions.html). For releases v0.0.6 and earlier, see the [MOLT repository](https://github.com/cockroachdb/molt/releases).
+
+Rename the binary to `molt` and add it to your `PATH` so you can execute the `molt fetch` command from any shell.
+
+## Setup
+
+Complete the following items before using MOLT Fetch:
+
+- Ensure that the source and target schemas are identical. Tables with mismatching columns may only be partially migrated.
+
+- Ensure that the SQL user running MOLT Fetch has the required privileges to run [`IMPORT INTO`]({% link {{ page.version.version }}/import-into.md %}#required-privileges) or [`COPY FROM`]({% link {{ page.version.version }}/copy-from.md %}#required-privileges) statements, depending on your intended [mode](#fetch-mode).
```

Comment:
I'm not sure where we wan't to call this bit out since its also part of setup, but if using AWS as cloud storage, you need to ensure the environment variables are exported on where you are running fetch from. AWS_REGION, AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY.

Similar for GCP, you need to ensure that gcloud auth has you signed in, and there are credentials in the default location for us to locate.